data/human/human missions.txt

```diff
@@ -5911,7 +5911,7 @@ mission "Quicksilver Mixup 3"
 		dialog `You have reached <planet>, but your escort carrying the misposted mail has not arrived. Better depart and wait for your escorts to arrive in this star system.`
 	on complete
 		payment 100000
-		dialog `The same manager from before is waiting with the unloading crew, and thanks you for covering for the other pilot before handing you the credit chip for payment. It's nice to know that captains who get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future."`
+		dialog `The same manager from before is waiting with the unloading crew, and thanks you for covering for the other pilot before handing you the credit chip for <payment>. It's nice to know that captains who get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future."`
```

Comment:
Actually, I think payment is not meant to be a substitution here. It literally just means payment here.

Reply:
It's kind of an incomplete sentence then. Or at least it doesn't make sense.

Reply:
I think the original payment is fine. Payment is a noun in itself, it's the same thing as saying "....handing you the credit chip for reimbursement". Or when you are handed your payment for a job irl; you're handed your "payment", not necessarily you are handed (insert the exact amount of money you made for that check). So it is indeed a complete sentence with a word that is used in the same manner all the time in real life.

Reply:
If the rest of the game does so using the substitution, then perhaps, but I'm fairly certain there are many places where the word itself is used.

Reply:
```suggestion
		dialog `The same manager from before is waiting with the unloading crew, and thanks you for covering for the other pilot before handing you the credit chip for payment. It's nice to know that captains who get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future."`
```